### PR TITLE
A few more compile-motivated tweaks

### PR DESCRIPTION
**Pull request by cmyr**
This is a rollup of three small commits fixing issues I encountered while hacking on feature compilation.

### CODE REVIEW

File: read-fonts/src/font_data.rs [hunk 1]
```diff
@@ -258,3 +258,12 @@ impl AsRef<[u8]> for FontData<'_> {
         self.bytes
     }
 }
+
+//kind of ugly, but makes FontData work with FontBuilder. If FontBuilder stops using
+//Cow in its API, we can probably get rid of this?
```
Comment (rsheeter) [inline read-fonts/src/font_data.rs:263]: We should perhaps create, and cite, an issue to not use Cow in the FontBuilder API?

File: write-fonts/src/font_builder.rs [hunk 2]
```diff
@@ -21,6 +21,11 @@ impl<'a> FontBuilder<'a> {
         self
     }
 
+    /// Returns `true` if the builder contains a table with this tag.
```
Comment (rsheeter) [inline write-fonts/src/font_builder.rs:24]: Comment adds nothing to type signature, suggest removal.

**Review by rsheeter — APPROVED** [review]
LGTM

File: read-fonts/src/font_data.rs (hunk 1 above)
Comment (cmyr) [reply read-fonts/src/font_data.rs:263]: #155

File: write-fonts/src/font_builder.rs (hunk 2 above)
Comment (cmyr) [reply write-fonts/src/font_builder.rs:24]: I think in general under-documentation is a far greater problem than over-documentation, and requiring the documentation of all public items is something that we can (and I think we should) do in CI. 🤷

**Pull request merged**